### DISCUSSION TITLE
Is anyone still needing JDK8 support?

### DISCUSSION THREAD

**Discussion by chrjohn** [quickfix-j/quickfixj#620] [Polls]
Just a quick poll if we could deprecate JDK 8 for QFJ 3.0.0 in favour of JDK11.
Don't know if we will actually do it, just curious what the opinion on this is.

**Comment by cd-ppi** [comment]
While we are currently moving our application to use Java 17, our customers don't follow along as fast as we would like them to do. So we would at least want (to avoid the term "need") security updates on a Java 8 compatible version for at least one and a half year, two and a half more likely.

**Comment by chrjohn** [comment]
@curdreinert No worries, of course there would be fixes for security problems for the last 2.3.x version. But please understand that I can only do this on a best effort basis.

Would be interesting to know from the people who still need JDK8 support: is it only because of the security updates?
